lib/Common/Memory/LargeHeapBlock.h

```diff
@@ -29,11 +29,11 @@ struct LargeObjectHeader
 #endif
 
 public:
-    bool markOnOOMRescan;
+    bool markOnOOMRescan:1;
+    bool isExplicitFreed:1;
+    bool isPageHeapAlloc:1;
```

Comment:
Heap block already has page heap mode- what do we need this for? What scenario do we have for storing this bit that we couldn't infer from the other flag?